compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -417,6 +472,50 @@ impl<'a> Parser<'a> {
                 return Ok(recovered);
             }
         }
+        if suggest_removing {
+            err.span_suggestion(
+                self.token.span,
+                format!(
+                    "Consider removing {}",
```

Comment:
```suggestion
                    "consider removing {}",
```

* https://rust-lang.github.io/rfcs/0344-conventions-galore.html#error-messages
* https://rustc-dev-guide.rust-lang.org/diagnostics.html#diagnostic-output-style-guide